config/locales/en.yml

```diff
@@ -29,8 +29,15 @@ en:
       calendar: "%a, %d %b %Y %H:%M:%S"
       usa: "%m/%d/%Y %l:%M %p"
 
+  Facility: "!activerecord.models.facility.one!"
+  Facilities: "!activerecord.models.facility.other!"
+  facility_downcase: facility
```

Comment:
Here's the meat of the technique. If I need capital facility, I can do something easy like `Manage !Facility!`.
For when it happens in longer sentences, I chose to go with `blah blah !facilities_downcase! blah blah`. It doesn't read quite as nicely, but I chose to go with this as opposed to needing to pass in `Facility.model_name.human` everywhere we need it. I couldn't use plain `facilities` because that's a later key in the yaml with children.